app/controllers/onboarding_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "#{Rails.root}/lib/alerts/onboarding_alert_manager"
```

Comment:
Use `require_dependency` instead of `require`. The latter can cause a lot of warnings in some circumstances.